homeassistant/components/esphome/config_flow.py

```diff
@@ -250,6 +251,48 @@ async def async_step_zeroconf(
 
         return await self.async_step_discovery_confirm()
 
+    async def async_step_mqtt(
+        self, discovery_info: MqttServiceInfo
+    ) -> ConfigFlowResult:
+        """Handle MQTT discovery."""
+        device_info: dict[str, Any] = json.loads(discovery_info.payload)
```

Comment:
Please use the `json_loads` helper

Reply:
`from homeassistant.util.json import json_loads`

Reply:
done